.github/CODEOWNERS

```diff
@@ -3,379 +3,596 @@
 
 # It uses the same pattern rule for gitignore file
 # https://git-scm.com/docs/gitignore#_pattern_format
+# Notably, a later match overrides earlier matches, so order matters.
+# If using a wildcard pattern, try to be as specific as possible to avoid
+# matching unintended files or overriding previous entries.
+# To exclude a file from ownership, add a line with only the file.
+# See the exclusions section at the end of the file for examples.
+
+# =========
+# Structure
+# =========
+#
+# The CODEOWNERS file is organised by topic area.
+# Please add new entries in alphabetical order within the relevant section.
+# Top-level sections are:
+#
+# * Buildbots, Continuous Integration, and Testing
+# * Build System
+# * Interpreter Core
+# * Documentation
+# * Other / Misc (Tools, Programs, Integration, etc)
+# * Platform Support
+# * Standard Library
```

Comment:
Having a short description for each entry might be useful, in particular for Documentation (which is mostly Sphinx-related files -- not stdlib modules docs), Standard Library (that includes both `Lib` and `Modules`, together with their tests and docs).

The description could be added at the beginning of each section (just after the header) instead.

I would also move Interpreter Core just after Platform Support.  As I see it, Interpreter Core and Standard Library include most of the entries, whereas the other sections include less entries that are specific to certain files.  This could also be stated in the comment above.

Reply:
Added & moved